src/Entity/PhoneNumberSettings.php

```diff
@@ -56,7 +56,7 @@ class PhoneNumberSettings extends ConfigEntityBase implements PhoneNumberSetting
    *
    * @var string
    */
-  protected $entity_type;
+  protected $entityType;
```

Comment:
Drupal uses underscore case for properties of entities. Typically camelCase is mostly found on private/protected properties or injected services of classes.
This should be reverted.